### PR TITLE
Added basic expression converter.

### PR DESCRIPTION
**Pull request by Dimach**

### CODE REVIEW

File: j2k/newSrc/org/jetbrains/kotlin/j2k/JavaToJKTreeBuilder.kt [hunk 1]
```diff
@@ -24,21 +24,68 @@ import org.jetbrains.kotlin.j2k.tree.impl.*
...
+                    return toJK()
+                }
+                is PsiLiteralExpression -> {
+                    return toJK()
+                }
+                is PsiCallExpression -> {
+                    return toJK()
+                }
+            }
+            throw RuntimeException("Not supported")
```
Comment (semoro) [inline j2k/newSrc/org/jetbrains/kotlin/j2k/JavaToJKTreeBuilder.kt]: use `throw UnsupportedOperationException("Not supported")` if it's permament, or `TODO` if not

File: j2k/newSrc/org/jetbrains/kotlin/j2k/JavaToJKTreeBuilder.kt [hunk 2]
```diff
@@ -24,21 +24,68 @@ import org.jetbrains.kotlin.j2k.tree.impl.*
...
+        fun PsiBinaryExpression.toJK(): JKExpression {
+            return JKBinaryExpressionImpl(lOperand.toJK(), rOperand?.toJK(), operationSign.toJK())
+        }
 
-                when (expression.literalElementType) {
-                    JavaTokenType.STRING_LITERAL ->
-                        resultExpression = JKJavaStringLiteralExpressionImpl(expression.innerText!!)
-                }
+        fun PsiLiteralExpression.toJK(): JKExpression {
+            if (this !is PsiLiteralExpressionImpl) {
```
Comment (semoro) [inline j2k/newSrc/org/jetbrains/kotlin/j2k/JavaToJKTreeBuilder.kt]: Assert here is better

Comment (semoro) [reply j2k/newSrc/org/jetbrains/kotlin/j2k/JavaToJKTreeBuilder.kt]: But it will not work for Kotlin without effect system, so no problem here

**Review by semoro — COMMENTED** [review]
Clean for now

**Review by Knize — APPROVED** [review]

**Review by semoro — CHANGES_REQUESTED** [review]
Please rebase onto latest nj2k branch and regenerate transformers

**Review by semoro — APPROVED** [review]

**Pull request merged**